LibreNMS/Validations/Programs.php

```diff
@@ -97,6 +97,7 @@ public function extraFpingChecks(Validator $validator, $cmd)
 
     private function failFping($validator, $cmd, $output)
     {
+        $validator->warn("fping FAILURES can be ignored if running LibreNMS in a jail without ::1. You may want to test it manually: fping ::1");
```

Comment:
this should be info()

Reply:
I'm stuck. I committed https://github.com/dlangille/librenms/commit/f6684161c31f89a288e8ea104d74ea5efd1802d1 and now I can't see how to push it here.

Reply:
you commit in the same branch and it happens automatically

Reply:
Did that, and the automatic part did not happen.

Reply:
@dlangille your last commit changes it back to warn from info.  Either revert that or just make a new commit changing it to info.

Reply:
Ouch. Repaired.